tests/src/main/java/com/linkedin/kmf/tests/BasicEndToEndTest.java

```diff
@@ -82,122 +83,122 @@ public void awaitShutdown() {
 
   /** Get the command-line argument parser. */
   private static ArgumentParser argParser() {
-    ArgumentParser parser = ArgumentParsers
-      .newArgumentParser("")
-      .defaultHelp(true)
-      .description("");
-
-    parser.addArgument("--topic")
-      .action(store())
-      .required(false)
-      .type(String.class)
-      .metavar("TOPIC")
-      .dest("topic")
-      .help("Produce messages to this topic and consume message from this topic");
-
-    parser.addArgument("--producer-id")
-      .action(store())
-      .required(false)
-      .type(String.class)
-      .dest("producerId")
-      .help("The producerId will be used by producer client and encoded in the messages to the topic");
+    ArgumentParser parser = ArgumentParsers.newArgumentParser("").defaultHelp(true).description("");
```

Comment:
Is there any reason to put them on the same line? Can we keep the same format as other `parser.addArgument(...)`?